packages/hardware-trezor/src/transformers/tx.ts

```diff
@@ -28,6 +28,7 @@ const trezorTxTransformer = async (
     networkId: context.chainId.networkId,
     outputs: mapTxOuts(body.outputs, context),
     protocolMagic: context.chainId.networkMagic,
+    referenceInputs: body.referenceInputs ? mapReferenceInputs(body.referenceInputs) : undefined,
```

Comment:
Reference inputs seem to have the same format as regular inputs. Is there a reason to duplicate the mapper?
```suggestion
    referenceInputs: body.referenceInputs ? mapTxIns(body.referenceInputs) : undefined,
```

Reply:
Thanks @mkazlauskas 👍 
I was also thinking about that. But, then I wanted to isolate it to have a specific output type declaration `CardanoReferenceInput` since `connect/trezor` package is changing a lot and we often face some breaking changes. Also, it is isolated in `hardware-ledger`  (https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/hardware-ledger/src/transformers/referenceInputs.ts) so I thought it is maybe a more correct way to do it. 

I'm happy to change it if you think it is better to use existing inputs mapper...?
cc @AngelCastilloB

Reply:
In CDDL it is the same type, so chance of it diverging is probably low
```
0 : set<transaction_input>    ; inputs
? 13 : set<transaction_input> ; collateral inputs
? 18 : set<transaction_input> ; reference inputs; New
```

Reply:
👌 then I will change it. Thanks a lot @mkazlauskas

Reply:
Changed in: [05495d6](https://github.com/input-output-hk/cardano-js-sdk/pull/995/commits/05495d6d7ed2e19d65a137a92dae0fd409c43d58)